src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java

```diff
@@ -179,7 +179,7 @@ public boolean isLightweight() {
             }
 
             FilePath scriptFile = dir.child(expandedScriptPath);
-            if (!new File(scriptFile.getRemote()).getCanonicalFile().toPath().startsWith(dir.absolutize().getRemote())) { // TODO JENKINS-26838
+            if (!new File(scriptFile.getRemote()).getCanonicalFile().toPath().startsWith(new File(dir.getRemote()).getCanonicalPath())) { // TODO JENKINS-26838
```

Comment:
Did you mean
```suggestion
            if (!new File(scriptFile.getRemote()).getCanonicalFile().toPath().startsWith(new File(dir.getRemote()).getCanonicalFile().toPath())) { // TODO JENKINS-26838
```
?

Reply:
or https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/139#discussion_r809047683

Reply:
Discussed in https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/139#discussion_r809043590.